pgdb/v1/fts.go

```diff
@@ -108,6 +106,40 @@ func FullTextSearchVectors(docs []*SearchContent, additionalFilters ...jargon.Fi
 			_ = err
 		}
 	}
+	return rv
+}
+
+func camelSplitDocs(docs []*SearchContent) []lexeme {
+	rv := make([]lexeme, 0, 8)
+	camelCaseRegex := regexp.MustCompile(`[[:upper:]][[:lower:]]+`)
+	acronymRegex := regexp.MustCompile(`(?P<token>[[:upper:]]+)([[:upper:]][[:lower:]]|$|\s)`)
```

Comment:
Please add a comment for this regex.

Reply:
Yes adding a comment here is a good idea, added a comprehensive comment in a2879af01d1254486a05a802fc1de351c5afb58d